src/main/java/com/neovisionaries/ws/client/WebSocketFrame.java

```diff
@@ -1199,15 +1198,17 @@ else if (frame.isContinuationFrame() == false)
 
         // Generate the first frame using the existing WebSocketFrame instance.
         // Note that the reserved bit 1 and the opcode are untouched.
-        byte[] payload = Arrays.copyOf(originalPayload, maxPayloadSize);
```

Comment:
`Arrays.copyOf()` is not available in Java 1.5 that is targeted by this library.